providers/defltprov.c

```diff
@@ -20,7 +20,7 @@
 #include "prov/provider_util.h"
 #include "internal/nelem.h"
 
-#define ALGC(NAMES, FUNC, CHECK) { { NAMES, "default=yes", FUNC }, CHECK }
+#define ALGC(NAMES, FUNC, CHECK) { { NAMES, "provider=default", FUNC }, CHECK }
```

Comment:
Is this syntactic sugar/shorthand really necessary?

Reply:
Yes. I have an upcoming script that needs something like this.